src/main/java/Application/OpenCVClient/Dockerfile

```diff
@@ -0,0 +1,21 @@
+#move to parent to use
+FROM ubuntu:16.04
+
+#setup tools
+COPY install.sh install.sh
+COPY Resources/opencv-450.jar /gen/opencv-450.jar
+COPY Resources/libopencv_java450.so /gen/libopencv_java450.so
+COPY Resources/singlemarkerssource.png /singlemarkerssource.png
+RUN chmod +x install.sh && sync && ./install.sh && rm install.sh
+
+#copy and build codes
+COPY build_classes.sh build_classes.sh
+COPY OpenCVClient/*.java /
+COPY Utilities/*.java /
+WORKDIR /
+RUN chmod +x build_classes.sh && sync && ./build_classes.sh && rm build_classes.sh
+
+#execute
+COPY OpenCVClient/client_exe.sh client_exe.sh
+WORKDIR /
+RUN chmod +x client_exe.sh && sync && ./client_exe.sh && rm client_exe.sh
```

Comment:
In AdvantEDGE, the last line will be commented and written in another interface, this line must have the required parameters, what you are doing here is writing the parameters inside a file (client_exe.sh), which means that we cannot modify them just by modifying this line.

What you can do is to expose these parameters as parameters for client_exe.sh, so this line would be:
`RUN chmod +x client_exe.sh && sync && ./client_exe.sh && rm client_exe.sh CLIENT_ID=client13 MASTER_URL=http://host.docker.internal:4567/ BEAT_COMMAND=client/register GET_SERVER_COMMAND=client/get_node/`
This way we can use it with AdvantEDGE.

And do the same with the server.